index.bs

```diff
@@ -1373,6 +1385,16 @@ To <dfn for="ReceiveStream">cancel</dfn> a {{ReceiveStream}} |stream|, run these
 1. Let |transport| be |stream|'s [=ReceiveStream/[[Transport]]=].
 1. Let |internalStream| be |stream|'s [=ReceiveStream/[[InternalStream]]=].
 1. Let |promise| be a new promise.
+1. Let |code| be 0.
+1. If |reason| is a {{WebTransportError}} and |reasons|'s [=[[ApplicationProtocolCode]]=] is not
+   null, then set |code| to |reason|'s [=[[ApplicationProtocolCode]]=].
+1. If |code| < 0, then set |code| to 0.
+1. If |code| > 255, then set |code| to 255.
+
+Note: |code| ranges in 0 and 255. The code will be encoded to a number in
+      [0x52e4a40fa8db, 0x52e4a40fa9e2] as decribed in [[WEB-TRANSPORT-HTTP3]].
+
+1. [=set/Remove=] |stream| from |transport|'s [=[[SendStreams]]=].
 1. Return |promise| and run the remaining steps [=in parallel=].
 1. [=Send STOP_SENDING=] with |internalStream|.
```

Comment:
I think this should become `[=Send STOP_SENDING=] with |internalStream| and |code|.`

Reply:
Done.